tasks/modprobe.yml

```diff
@@ -1,3 +1,14 @@
 ---
 
-# modprobe
+- name: install modprobe to disable filesystems | os-10
+  package:
+    name: 'kmod'
+    state: 'installed'
+
+- name: disable unused filesystems | os-10
+  template:
+    src: "modprobe.j2"
+    dest: "/etc/modprobe.d/dev-sec.conf"
+    owner: "root"
+    group: "root"
+    mode: "0640"
```

Comment:
Seems your editor did not add a `\n` automatically. You can check with a hexeditor. Can you add one just to be safe :) ?